src/main/java/org/spongepowered/common/data/processor/data/entity/TagsDataProcessor.java

```diff
@@ -63,8 +62,7 @@ protected boolean set(Entity dataHolder, Set<String> value) {
         Set<String> added = new HashSet<String>();
```

Comment:
You can just use a list here as you do not need a set to guarantee. Moreover, if you insist using a `Set`, please use a `LinkedHashSet` which has a better iteration performance.